internal/httputil/request.go

```diff
@@ -2,14 +2,17 @@ package httputil
 
 import "net/http"
 
-// IsTransferEncodingChunked checks that a http.Request is not
-// chunked. Requests with this transfer encoding do not include
-// a valid Content-Length header.
-func IsTransferEncodingChunked(req *http.Request) bool {
+// TransferEncoding gets the header value from the request.
```

Comment:
are we using these funcs anywhere other than tests?

Reply:
no, internal